charts/catalog/README.md

```diff
@@ -48,6 +48,10 @@ chart and their default values.
 | `apiserver.storage.type` | The storage backend to use; the only valid value is `etcd`, left for other storages support in future, e.g. `crd` | `etcd` |
 | `apiserver.storage.etcd.useEmbedded` | If storage type is `etcd`: Whether to embed an etcd container in the apiserver pod; THIS IS INADEQUATE FOR PRODUCTION USE! | `true` |
 | `apiserver.storage.etcd.servers` | If storage type is `etcd`: etcd URL(s); override this if NOT using embedded etcd | `http://localhost:2379` |
+| `apiserver.storage.etcd.persistence.enabled` | Enable persistence using PVC | `false` |
```

Comment:
Is this the minimal thing use a `--set` on to enable this? 

```
--set apiserver.storage.etcd.persistence.enabled=true
```

Reply:
Yes.